Dockerfile

```diff
@@ -1,14 +1,21 @@
 FROM gcr.io/distroless/static-debian12:latest AS build
 
-FROM scratch
-# needed for version check HTTPS request
+ARG TARGETPLATFORM
+FROM --platform=$TARGETPLATFORM alpine:3.18 AS security_provider
+RUN addgroup -S nonroot && adduser -S nonroot -G nonroot
+
+FROM gcr.io/distroless/base-debian12
```

Comment:
I don't think there is a need for multiple stages anymore, we can use the static variant directly.